scripts/test/SANS/algorithm_detail/move_workspaces_test.py

```diff
@@ -0,0 +1,256 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+
+import unittest
+
+import mantid.simpleapi
+from sans.algorithm_detail.move_workspaces import SANSMoveZOOM, SANSMoveSANS2D
+from sans.common.enums import SANSFacility, SANSInstrument, DetectorType
+from sans.state.data import get_data_builder
+from sans.state.move import StateMoveZOOM, get_move_builder
+from sans.test_helper.file_information_mock import SANSFileInformationMock
+
+
+def get_coordinates():
+    # Co-ords are used for low angle bank movements
+    return [0, 0]
+
+
+def get_monitor_pos(ws, monitor_spectrum_no, move_info):
+    monitor_name = move_info.monitor_names[str(monitor_spectrum_no)]
+    instrument = ws.getInstrument()
```

Comment:
@DavidFair  you are using the old access methods which should be treated as deprecated. Do not call `getInstrument` in your code. Use `instrument 2.0` instead.

For example
```python
comp_info = ws.componentInfo()
return comp_info.position(comp_info.indexOfAny(monitor_name))
```

Reply:
https://docs.mantidproject.org/nightly/concepts/InstrumentAccessLayers.html

Reply:
Thanks Owen, I've addressed this in the code I've touched move_workspaces

There is one place where `getInstrument` is still called. Currently, a base instrument is used for the original positions. What would be the direct replacement of:

```
    instrument = workspace.getInstrument()
    base_instrument = instrument.getBaseInstrument()
```

Reply:
Instrument 2 doesn't have a direct concept of base instrument. Copies are first-class. What exactly do you need the base instrument for, maybe I can help more specifically then.

Reply:
I don't think we've ever done this from the python side, but if it's possible to get hold of the workspace in it's "base", unaltered configuration then we should be able to do this.

Reply:
I've only recently started working on this codebase but my naive understanding is that we are basically using a base instrument to reset any offset positions back to the original IDF positions.

Would it be worth opening a maintenance issue to address this, as I feel it falls out of the scope of the PR. However, it's well worth looking into, as there is a code smell if you've not seen this before.

Reply:
As a side note - It's good that developers are better aware of what instrument 2.0 offers, i've seen a number of developers (including senior ones) reverting to the old ways of working. Please pass on the knowledge as part of your reviews of other issues.